app/default_overrides.go

```diff
@@ -75,12 +79,33 @@ func (stakingModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
 	params := stakingtypes.DefaultParams()
 	params.UnbondingTime = appconsts.DefaultUnbondingTime
 	params.BondDenom = BondDenom
+	params.MinCommissionRate = sdk.NewDecWithPrec(5, 2) // 5%
 
 	return cdc.MustMarshalJSON(&stakingtypes.GenesisState{
 		Params: params,
 	})
 }
 
+// stakingModule wraps the x/staking module in order to overwrite specific
+// ModuleManager APIs.
+type slashingModule struct {
+	slashing.AppModuleBasic
+}
+
+// DefaultGenesis returns custom x/staking module genesis state.
+func (slashingModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
+	params := slashingtypes.DefaultParams()
+	params.MinSignedPerWindow = sdk.NewDecWithPrec(5, 2) // 5%
```

Comment:
[nit] should the specs state `5%`?

https://github.com/celestiaorg/celestia-app/blob/4e62d2d3660e8ad5b6115348bed5aa6a57cf93fc/specs/src/specs/params.md?plain=1#L45 be

Reply:
update to 75% across both